From: Ben Hutchings <ben@decadent.org.uk>
Subject: udl: Disable autoload
Date: Sun, 23 Dec 2012 23:58:16 +0000

This kernel driver is incompatible with older versions of
xserver-xorg-video-displaylink, so do not autoload it yet.  It will
have to be loaded either by local configuration or as part of a
package of a compatible X driver.

--- a/drivers/gpu/drm/udl/udl_drv.c
+++ b/drivers/gpu/drm/udl/udl_drv.c
@@ -30,7 +30,7 @@ static struct usb_device_id id_table[] =
 			USB_DEVICE_ID_MATCH_INT_PROTOCOL,},
 	{},
 };
-MODULE_DEVICE_TABLE(usb, id_table);
+/*MODULE_DEVICE_TABLE(usb, id_table);*/
 
 MODULE_LICENSE("GPL");
 
